Library/.rubocop.yml

```diff
@@ -6,6 +6,12 @@ AllCops:
     - '**/Casks/**/*'
     - '**/vendor/**/*'
 
+require:
+  - ./Homebrew/rubocops/bottle_block_cop.rb
```

Comment:
Let's just hard-code all the requires for now; dynamic requiring is a bit overkill for now.